src/main/java/net/glowstone/entity/objects/GlowItemFrame.java

```diff
@@ -196,10 +200,10 @@ public void setFacingDirection(BlockFace blockface) {
 
     @Override
     public void setItem(ItemStack is) {
-        if (is == null) {
-            is = new ItemStack(Material.AIR, 1);
-        }
+        is = InventoryUtil.itemOrEmpty(is);
+        is = is.clone();
         is.setAmount(1);
```

Comment:
Alright. You can still simplify this with `is = InventoryUtil.itemOrEmpty(is).clone()`.